<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditExistingPartitionDialog</class>
 <widget class="QDialog" name="EditExistingPartitionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>203</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Edit Existing Partition</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Si&amp;ze:</string>
       </property>
       <property name="buddy">
        <cstring>sizeSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="PartitionSizeWidget" name="sizeSpinBox">
       <property name="suffix">
        <string> MB</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Content:</string>
       </property>
       <property name="buddy">
        <cstring>keepRadioButton</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="keepRadioButton">
       <property name="text">
        <string>Keep</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="formatRadioButton">
       <property name="text">
        <string>Format</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Warning: Formatting the partition will erase all existing data.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="mountPointLabel">
       <property name="text">
        <string>&amp;Mount Point:</string>
       </property>
       <property name="buddy">
        <cstring>mountPointComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="mountPointComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <item>
        <property name="text">
         <string>/</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>/boot</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>/home</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>/opt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>/usr</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>/var</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PartitionSizeWidget</class>
   <extends>QSpinBox</extends>
   <header location="global">PartitionSizeWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sizeSpinBox</tabstop>
  <tabstop>keepRadioButton</tabstop>
  <tabstop>formatRadioButton</tabstop>
  <tabstop>mountPointComboBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditExistingPartitionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditExistingPartitionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
